src/components/atoms/ControlledFields/DateTimePicker/DateTimePicker.tsx

```diff
@@ -0,0 +1,64 @@
+import { FormHelperText, Typography } from '@mui/material';
+import Box from '@mui/material/Box';
+// import { DateTimePicker as MuiDateTimePicker } from '@mui/x-date-pickers/DateTimePicker';
+// import dayjs from 'dayjs';
+import { useState } from 'react';
+import { useController } from 'react-hook-form';
+
+// import { CalendarIcon } from '~/components/atoms';
+import { InputLabel } from '../InputLabel';
+import { IProps } from './types';
+
+export const DateTimePicker = <T extends object>({
+  control,
+  name,
+  label,
+  // minDate,
+  helperText = '',
+}: IProps<T>) => {
+  const {
+    field,
+    fieldState: { error },
+  } = useController({
+    name,
+    control,
+  });
+
+  const errorMessage = error?.message;
+
+  const [open, setOpen] = useState(false);
+
+  return (
+    <Box width="100%">
+      {label && (
+        <InputLabel htmlFor={name} error={!!errorMessage}>
+          {label}
+        </InputLabel>
+      )}
+      {/* <MuiDateTimePicker
```

Comment:
commented code

Reply:
I commented this code in order not to install an additional package for dates manipulations.

Reply:
Aaaaa, ok 👍  
just leave the comment about that at the top of the file